content/concepts/payment-system-basics/ledgers.md

```diff
@@ -23,10 +23,28 @@ As its name might suggest, a ledger's state tree is a tree data structure. Each
 
 In the case of transactions, the identifying hash is based on the signed transaction instructions, but the contents of the transaction object when you look it up also contain the results and metadata of the transaction, which are not taken into account when generating the hash.
 
+## Open, Closed, and Validated Ledgers
+
+The `rippled` server makes a distinction between ledger versions that are _open_, _closed_, and _validated_. A server has one open ledger, a history of immutable, validated ledgers, and any number of closed but unvalidated ledgers. The following table summarizes the difference:
```

Comment:
So, closed ledgers are immutable too but I don't know if that's a distinction you care to make.

Reply:
Yeah, not sure if it's helpful or confusing to clarify that validated ledgers are a subset of closed ledgers... Since even though validated ledgers are immutable, they can be thrown out and replaced with alternate versions that have the same sequence number, which even though it fits the programming definition of immutability, may not meet some people's expectations of what "immutable" means...

Reply:
I would consider a ledger "mutated" if it is not included in the history of the latest fully validated ledger. I think it's OK to use this alternate meaning of "immutable" because we are saying that the ledger history is immutable. On the other hand, if we're concerned that "immutable" has multiple meanings, we could use one of these terms used by other public blockchains: `confirmed` or `finalized`.

Reply:
Suggested nit edit: Would it be helpful to change the order in which we describe the ledger types above to match the order in which the table below presents the ledger types? So instead of open, validated, and closed - open, closed, and validated?

Reply:
Changed the sentence in question to:

> A server has one open ledger, any number of closed but unvalidated ledgers, and an immutable history of closed and validated ledgers.